Source/Foundation/bsfCore/Audio/BsAudio.h

```diff
@@ -91,6 +91,15 @@ namespace bs
 		virtual SPtr<AudioClip> createClip(const SPtr<DataStream>& samples, UINT32 streamSize, UINT32 numSamples,
 			const AUDIO_CLIP_DESC& desc) = 0;
 
+		/** 
+		 * Creates a new audio clip.
+		 *
+		 * @param[in]	samples		Stream containing audio samples in format specified in @p desc.
+		 * @param[in]	desc		Descriptor describing the type of the audio stream (format, sample rate, etc.).
+		 * @return					Newly created AudioClip. Must be manually initialized.
+		 */
+		virtual SPtr<AudioClip> createClip(const SPtr<DataStream>& samples, const AUDIO_CLIP_DESC& desc) = 0;
```

Comment:
Documentation might note here it's for creating an endless audio clip. Alternatively we could maybe use the constructor above but assume endless if streamSize and numSamples is 0. Not sure if that's a better option though, probably doesn't matter much.